### PR TITLE
Blog post for NLGrant announcement

### PR DESCRIPTION
**Pull request by atbrakhi**

### CODE REVIEW

File: assets/img/blog/floats-clear.png
Comment (mrego) [inline assets/img/blog/floats-clear.png:1]: Super-nit: Maybe rename the file so it has a date too, so in the future we avoid conflicts with names.

File: assets/img/blog/floats.png
Comment (mrego) [inline assets/img/blog/floats.png:1]: Ditto about the filename.

File: _posts/2023-11-06-nlgrant-announcement.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout:     post
+tags:       blog
+title:      "Servo announces grant from the NLnet Foundation"
+date:       2023-11-06
+summary:    Earlier this year in July, Servo received a NLNet grant to enhance several aspects of Servo.
```
Comment (mrego) [inline _posts/2023-11-06-nlgrant-announcement.md:6]:
```suggestion
summary:    Earlier this year in July, Servo received a NLnet grant to enhance several aspects of Servo.
```

File: _posts/2023-11-06-nlgrant-announcement.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+categories:
+---
+
+We are excited to announce that earlier this year in July, Servo received a **[NLNet grant](https://nlnet.nl/project/Servo/)** to enhance several aspects of Servo. Under this grant, our primary focus is to:
```
Comment (mrego) [inline _posts/2023-11-06-nlgrant-announcement.md:10]:
```suggestion
We are excited to announce that earlier this year in July, Servo received a **[NLnet grant](https://nlnet.nl/project/Servo/)** to enhance several aspects of Servo. Under this grant, our primary focus is to:
```

File: _posts/2023-11-06-nlgrant-announcement.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+* **Complete float support in Servo**
+* **Support more languages in inline layout**
+* **Add initial &lt;table> support**
+
+<h2>Floats </h2>
```
Comment (mrego) [inline _posts/2023-11-06-nlgrant-announcement.md:16]:
```suggestion
## Floats
```

File: _posts/2023-11-06-nlgrant-announcement.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+Supporting floats in Servo is an ongoing effort since mid-2023. We’ve made significant progress on floats, but there are still some issues that need to be addressed before Servo can boast a fully-compliant implementation of CSS floats.
+
+Our objective is to achieve an average pass rate of over 80% for <code>/css/CSS2/floats/</code> and <code>/css/CSS2/floats-clear/</code>. You can track the results on our **[WPT dashboard](https://wpt.servo.org)**
+
+Last week, we surpassed this for the floats tests, with an 82.2% WPT pass rate:
+
+<img src="{{ '/img/blog/floats.png' | url }}" style="width: 60%; margin: 1em;" alt="wpt floats" />
```
Comment (mrego) [inline _posts/2023-11-06-nlgrant-announcement.md:24]: Please fix image alt text.

File: _posts/2023-11-06-nlgrant-announcement.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+We’re also nearing the milestone for floats-clear, currently at a 73.3% pass rate:
+
+<img src="{{ '/img/blog/floats-clear.png' | url }}" style="width: 60%; margin: 1em;" alt="wpt floats" />
+
+<h2>More languages in inline layout </h2>
```
Comment (mrego) [inline _posts/2023-11-06-nlgrant-announcement.md:30]:
```suggestion
## More languages in inline layout
```

File: _posts/2023-11-06-nlgrant-announcement.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+Our objective is to achieve an average pass rate of over 80% for <code>/css/CSS2/floats/</code> and <code>/css/CSS2/floats-clear/</code>. You can track the results on our **[WPT dashboard](https://wpt.servo.org)**
+
+Last week, we surpassed this for the floats tests, with an 82.2% WPT pass rate:
+
+<img src="{{ '/img/blog/floats.png' | url }}" style="width: 60%; margin: 1em;" alt="wpt floats" />
+
+We’re also nearing the milestone for floats-clear, currently at a 73.3% pass rate:
+
+<img src="{{ '/img/blog/floats-clear.png' | url }}" style="width: 60%; margin: 1em;" alt="wpt floats" />
```
Comment (mrego) [inline _posts/2023-11-06-nlgrant-announcement.md:28]: Ditto.

File: _posts/2023-11-06-nlgrant-announcement.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+<h2>More languages in inline layout </h2>
+
+Servo’s layout engine lacks crucial features for rendering languages that don’t use the Latin alphabet. This includes proper font selection, support for right-to-left scripts, and logical properties. Our aim is to improve Servo’s support for displaying a wider variety of content.
+
+<h2>Initial &lt;table> support </h2>
```
Comment (mrego) [inline _posts/2023-11-06-nlgrant-announcement.md:34]:
```suggestion
## Initial &lt;table> support
```

**Review by mrego — APPROVED** [review]
Thanks, looks good but we need to fix alt text for images. The rest of things are mostly nits and minor suggestions.

**Comment by atbrakhi** [comment]
Thanks for the review, I have update the PR.

**Pull request merged**